### PR TITLE
Update github.com/google/go-github to v51

### PR DESCRIPTION
**Pull request by harryzcy**
`github.com/google/go-github` has new major version releases frequently. It is required to update all import path, in additional to `go.mod`

### CODE REVIEW

**Comment by KN4CK3R** [comment]
CI fail is related.

**Review by delvh — APPROVED** [review]
Yup, just checked their changelog, you didn't miss any breaking change.

**Comment by delvh** [comment]
> CI fail is related.

Well, now it isn't anymore

**Review by KN4CK3R — APPROVED** [review]

File: services/migrations/github.go [hunk 1]
```diff
@@ -259,11 +259,11 @@ func (g *GithubDownloaderV3) GetMilestones() ([]*base.Milestone, error) {
...
 				Title:       m.GetTitle(),
 				Description: m.GetDescription(),
-				Deadline:    m.DueOn,
+				Deadline:    &m.DueOn.Time,
 				State:       state,
-				Created:     m.GetCreatedAt(),
-				Updated:     m.UpdatedAt,
-				Closed:      m.ClosedAt,
+				Created:     m.GetCreatedAt().Time,
+				Updated:     &m.UpdatedAt.Time,
```
Comment (6543) [inline services/migrations/github.go]: is there no GetTime func ?

Comment (6543) [reply services/migrations/github.go]: nop

**Review by 6543 — APPROVED** [review]

**Comment by jolheiser** [comment]
It looks like the CI failures may still be related. 🙂

**Comment by harryzcy** [comment]
> It looks like the CI failures may still be related. 🙂

Ok, I'm trying to reproduce it locally. But it seems the failing check is not running `because GITHUB_READ_TOKEN is empty`. 

What is `GITHUB_READ_TOKEN` though? Is it any github token with read access?

**Comment by 6543** [comment]
well I created a pull upstream https://github.com/google/go-github/pull/2743 so we can use that later ...

**Pull request merged**